packages/design-system/src/Toast.stories.tsx

```
@@ -1,152 +1,170 @@
+            description:
+              'Dolore incididunt reprehenderit officia et occaecat non laboris.',
+          })
+        }
+      >
+        Trigger
+      </Button>
+    </>
   )
 }
 
 export const Error = () => {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)